### PR TITLE
Adding Girls Who Code to Organizations

### PR DESCRIPTION
**Pull request by erikiva**

### CODE REVIEW

**Comment by dreamingechoes** [comment]
Thank you so much for this contribution! :sparkling_heart: Before merge the PR, don’t you want to add yourself to the contributors list?

**Comment by erikiva** [comment]
Just realised the commit comment is wrong :P How do I do it, add myself and include the commit  for the PR?

File: README.md [hunk 1]
```diff
@@ -35,6 +36,10 @@ Organizations bringing diversity to tech.
 - [Women Techmakers](https://www.womentechmakers.com/) - Google's program that provides visibility, community, and resources for women in technology.
 - [YesWeTech](http://yeswetech.org/) - Feminist community working for an egalitarian and proud culture of women interested in technology.
 
+## Conferences
+The following is a list of conferences that make a special effort to be inclusive and encourage diversity
+
+- [Elixir With Love] (https://www.elixir-with-love.com/)
```
Comment (dreamingechoes) [inline README.md]: There’s a whitespace here that it’s breaking the markdown formating, maybe we have to fix it before merge the PR :smile:

**Comment by dreamingechoes** [comment]
Superb! I’ll merge this right now! Again, thank you for this awesome contribution 🤗

**Pull request merged**